CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+
+### v0.35.0
```

Comment:
I'm sure we've discussed it, but should this be `1.0`?

In favor of `1.0`:
- This is a _complete rewrite._ Feels weird to bump a minor version when it's a ground-up rewrite!
- According to semver, this is a breaking change to APIs and therefore should be a major version bump.
- If someone has `'>=0.34'` in there `package.json`, they'll get updated to `0.35` (I think that's how it works if you haven't specified a patch version) and stuff will break.

In favor of `0.35`:
- We wouldn't want to indicate we think things are production-ready by slapping a 1.0 on this.
- Maybe we want some sort of marketing around a 1.0 release and want to save it?
- Perhaps we're saving 1.0 for the Rust WASM rewrite?
- Prior to 1.0, it's expected that many changes will be breaking changes! So no worries about violating semver.

Curious what others think.

Reply:
Talked to @therealjeffg about it today and he's in favor of `0.35` as we don't want to indicate production-readiness with the `1.0` label.

Marking this as resolved!

Reply:
In reaction to this comment:

> If someone has '>=0.34' in there package.json, they'll get updated to 0.35 (I think that's how it works if you haven't specified a patch version) and stuff will break.

I agree with this sentiment - I think we'll get away with it this time because we don't have a lot of downstream adoption currently but in the future we need to be much better wrt helping developers consuming the library stay up to date. This isn't at the standard we should strive for, for exactly the reason Jess points out.

Reply:
Just wanted to this here: Normally people will install webnative using `npm install`, and that'll set the version to `^0.34.2`. That means people **will not** automatically get `0.35.0`, because the version starts with a zero, according to https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004